test/test_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'left_joins'
 
 require 'minitest/autorun'
+Minitest::Test = MiniTest::Unit::TestCase unless defined? Minitest::Test
```

Comment:
Note: this line is needed because of the different naming between minitest 4.2 and 5.0.
And ActiveSupport 4.0 has dependency: `minitest ~> 4.2` while ActiveSupport 4.1 has dependency: `minitest ~> 5.1`